go.mod

```diff
@@ -5,6 +5,7 @@ go 1.20
 require (
 	github.com/go-ldap/ldap/v3 v3.4.6
 	github.com/notaryproject/notation-core-go v1.0.1
+	github.com/notaryproject/notation-plugin-framework-go v0.0.0-20240103032027-c077edacd1ef
```

Comment:
Yeah, I am planning to release `notation-plugin-framework-go` once this PR is ready to be merged(wont merge it until we release `notation-plugin-framework-go`.

Reply:
They are `golang.org/x` packages and I think they will never be stabilized. The reason is simple that once they are stabilized, they will be moved to the golang built-in libraries.

> These repositories are part of the Go Project but outside the main Go tree. They are developed under looser compatibility requirements than the Go core.

Reply:
Basically, since `notation-go` depends on `notation-plugin-framework-go` and expose `notation-plugin-framework-go` as part of its API. The instability of `notation-plugin-framework-go` makes `notation-go` instable. However, `notation-go` does not expose `golang.org/x` packages as part of its API.

Reply:
> Basically, since `notation-go` depends on `notation-plugin-framework-go` and expose `notation-plugin-framework-go` as part of its API.

I find a solution. How about we keep everything under https://github.com/notaryproject/notation-go/tree/main/plugin and mark all of them `Deprecated`. Meanwhile, we replace the implementation of `signer/plugin.go` and `verifier/verifier.go` with `notation-plugin-framework-go` as you've done in this PR?

Reply:
umm I think we are not going to make any breaking changes in `notation-plugin-framework-go` so after next planned release (this week?) we can take dependency on it and before before next to next release, we will release `notation-plugin-framework-go-1.0`. Does that sounds fine? Also, we have enough unit/tests to avoid any breaking change in notation-go.

PS: I wont merge this PR until next release,

Reply:
Updated `go.mod` to depend upon notation-plugin-framework-go-1.0